SDK/simUtil.h

```diff
@@ -48,13 +48,12 @@
 #include "simUtil/Replaceables.h"
 #include "simUtil/ResizeViewManipulator.h"
 #include "simUtil/ScreenCoordinateCalculator.h"
-#include "simUtil/Shaders.h"
-#include "simUtil/SilverLiningSettings.h"
+#include "simUtil/SilverLining.h"
 #include "simUtil/StatsHandler.h"
 #include "simUtil/StatsSizeFixer.h"
 #include "simUtil/StatusText.h"
 #include "simUtil/TerrainToggleEffect.h"
-#include "simUtil/TritonSettings.h"
+#include "simUtil/Triton.h"
```

Comment:
Definitely would need to update genTopLevelHeaders.sh to account for this so it doesn't get lost in the future.

If the simUtil.h file was removed as I'm thinking, then this extra work for the Triton.h/SilverLining.h would not be required.